cns/restserver/homeazmonitor.go

```diff
@@ -10,13 +10,14 @@ import (
 	"github.com/Azure/azure-container-networking/cns/logger"
 	"github.com/Azure/azure-container-networking/cns/types"
 	"github.com/Azure/azure-container-networking/nmagent"
+	"github.com/google/go-cmp/cmp"
```

Comment:
[do not use cmp](https://pkg.go.dev/github.com/google/go-cmp/cmp)
> It is intended to only be used in tests, as performance is not a goal and it may panic if it cannot compare the values. Its propensity towards panicking means that its unsuitable for production environments where a spurious panic may be fatal.